common/scripts/poll-npm-package-published.mjs

```diff
@@ -0,0 +1,57 @@
+import { get } from "https";
+import process from "process";
+
+/**
+ * Script that polls the npm registry to see if a @azure/communication-react package version published.
+ * By default, it polls on short intervals (5 seconds) and timesout after a longer interval (5 minutes).
+ *
+ * @params {string} version - the version to check for
+ *
+ * @example node ./poll-npm-package-published.js 1.3.0
+ */
+
+const POLL_INTERVAL_MS = 5000; // 5 seconds in millis
+const TIMEOUT_TIME_MS = 12 * 1000; // 5 minutes in millis
+const GET_REQUEST_TIMEOUT_MS = 5000;
+
+const GET_REQUEST_OPTIONS = {
+  hostname: 'registry.npmjs.org',
+  timeout: GET_REQUEST_TIMEOUT_MS
+}
+
+const checkNpm = (packageName, packageVersion) => {
+  return new Promise((resolve, reject) => {
+    const request = get({...GET_REQUEST_OPTIONS, path: `/${packageName}/${packageVersion}`}, (response) => {
+      resolve(response);
+    });
+    console.log(`Pinging: https://${GET_REQUEST_OPTIONS.hostname}/${packageName}/${packageVersion}`);
+    request.end();
+  });
+}
+
+const sleep = (ms) => new Promise( res => setTimeout(res, ms));
+
+const main = async () => {
+  const packageVersion = process.argv[2];
+  const packageName = `@azure/communication-react`
+  const startTime = new Date();
+  while(true) {
+    const response = await checkNpm(packageName, packageVersion);
+    console.log('responseCode: ', response.statusCode);
+
+    if (response.statusCode === 200) {
+      console.log("Successfully found npm package")
+      process.exitCode = 0;
+      return;
+    } else if ((new Date() - startTime) > TIMEOUT_TIME_MS) {
+      console.log('Failed to find package on the npm registry');
+      process.exitCode = 1;
+      return;
+    } else {
+      console.log('sleeping for 5 seconds...');
```

Comment:
nit: I'd just say `Sleeping for a bit...` so that you don't have to update this message if you update `POLL_INTERVAL_MS` (or more likely, we'll forget to update this comment and then it will be wrong logging (which is worse than no logging at all)